src/as-system-info.c

```diff
@@ -49,8 +49,15 @@
 
 #if defined(__linux__)
 #include <sys/sysinfo.h>
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__APPLE__)
 #include <sys/types.h>
+# ifdef __APPLE__
```

Comment:
```suggestion
#ifdef __APPLE__
```
nit: no space after `#` please